specs/error_state/ErrorInvalidCreationCode.md

```diff
@@ -0,0 +1,37 @@
+# ErrorInvalidCreationCode state
+
+## Procedure
+
+InvalidCreationCode is an error related with code store. This error only occurs when executing `CREATE` or `CREATE2` opcode or contract creation transaction (`tx.to == null`).
+
+### EVM behavior
+
+When handling a contract creation transaction, initial bytecode are executed and the current call context is `CREATE` or `CREATE2`. The last opcode to store for new contract is `RETURN`. It returns `memory[offset...offset + length]` content as contract bytecode to store into state DB.
```

Comment:
This is similar description as [here](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/475#discussion_r1280377290) where @rrtoledo suggested an improved text. Might be applied here too :).

Reply:
fixed in [4a3594a](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/478/commits/4a3594a0864cbb188e223aa433451b3fabc3f4b8)